File: src/main/java/com/example/mod/listener/RenderGameOverlayEventListener.java
package com.example.mod.listener;

import net.weavemc.api.RenderGameOverlayEvent;
import net.weavemc.api.event.SubscribeEvent;

public class RenderGameOverlayEventListener {
    @SubscribeEvent
    public void onEvent(RenderGameOverlayEvent event) {
        System.out.println("Render overlay event");
    }
}

File: src/main/kotlin/com/example/mod/listener/RenderGameOverlayEventListener.kt
package com.example.mod.listener

import net.weavemc.api.RenderGameOverlayEvent
import net.weavemc.api.event.SubscribeEvent

class RenderGameOverlayEventListener {
    @SubscribeEvent
    fun onEvent(event: RenderGameOverlayEvent) {
        println("Render overlay event")
    }
}
